backend/src/main/kotlin/blue/mild/covid/vaxx/setup/ConfigurationDependencyInjection.kt

```diff
@@ -31,6 +33,16 @@ private fun loadVersion(defaultVersion: String = "development"): String = runCat
 // TODO load all config from the file and then allow the replacement with env variables
 fun DI.MainBuilder.bindConfiguration() {
 
+    val apiKey = getEnv("MAIL_JET_API_KEY")
```

Comment:
co zkusit `requireNotNull(getEnv("MAIL_JET_API_KEY") { "MAIL_JET_API_KEY env variable was not provided. Exiting" }` -- mam dojem, ze by to mohlo spadnout a exitnout aplikaci